tm2/pkg/bft/privval/signer/remote/messages.go

```diff
@@ -0,0 +1,51 @@
+package remote
+
+import (
+	"github.com/gnolang/gno/tm2/pkg/crypto"
+)
+
+// MaxMessageSize is the maximum size of a message that can be sent or received.
+const MaxMessageSize = 1024
```

Comment:
Is this in bytes?

Reply:
Yes this it in bytes.

Reply:
Can we add a comment? 🙏

Reply:
Done https://github.com/gnolang/gno/pull/3554/commits/07f66dc69b3f5f341ed222bd95a94f7bdae6b3b7